### PR TITLE
Updating Brev.Dev documentation

### PR DESCRIPTION
**Pull request by polinabinder1**
Updated Readme/ipython notebooks with instructions on brev.dev usage.

### CODE REVIEW

File: sub-packages/bionemo-example_model/README.md [hunk 1]
```diff
@@ -4,9 +4,11 @@
...
 
-This tutorial demonstrates the creation of a simple MNIST model. This should be run in a BioNeMo container. For this tutorial, we will reuse elements from the BioNeMo example_model package.
+This tutorial demonstrates the creation of a simple MNIST model. This should be run in a BioNeMo container. The BioNeMo Framework container can run in a brev.dev launchable: [![ Click here to deploy.](https://uohmivykqgnnbiouffke.supabase.co/storage/v1/object/public/landingpage/brevdeploynavy.svg)]([https://console.brev.dev/launchable/deploy?launchableID=env-2pPDA4sJyTuFf3KsCv5KWRbuVlU](https://console.brev.dev/launchable/deploy/now?launchableID=env-2pPCVle3aYyGMbPfA61vCf1Au4X). It takes about 10 minutes to deploy this notebook as a Launchable. As of this writing, we are working on a free tier so a credit card may be required. You can reach out to your NVIDIA rep for credit. Notebooks and a shell interface can be launced by clicking `Open Notebook`. (Note: This links to the nightly release and may be out of date with these docs.)
```
Comment (jstjohn) [inline sub-packages/bionemo-example_model/README.md:7]: This link looks weird, you pasted it in nicely in the `access-startup.md` doc but not this one.

**Review by jstjohn — APPROVED** [review]

File: sub-packages/bionemo-example_model/README.md (hunk 1 above)
Comment (polinabinder1) [reply sub-packages/bionemo-example_model/README.md]: fixed

File: docs/docs/user-guide/examples/bionemo-esm2/inference.ipynb [hunk 2]
```diff
@@ -6,7 +6,7 @@
    "source": [
     "[![ Click here to deploy.](https://uohmivykqgnnbiouffke.supabase.co/storage/v1/object/public/landingpage/brevdeploynavy.svg)](https://console.brev.dev/launchable/deploy?launchableID=env-2p8Dz9VJ2kFSrTQMXVRtUMgymKx)\n",
     "\n",
-    "<div class=\"alert alert-block alert-info\"> <b>NOTE</b> It takes about 10 minutes to deploy this notebook as a Launchable. As of this writing, we are working on a free tier so a credit card may be required. You can reach out to your NVIDIA rep for credits.</div>"
+    "<div class=\"alert alert-block alert-info\"> <b>NOTE</b> It takes about 10 minutes to deploy this notebook as a Launchable. As of this writing, we are working on a free tier so a credit card may be required. You can reach out to your NVIDIA rep for credits. (Note: This links to the nightly release and may be out of date with these docs.) </div>"
```
Comment (tshimko-nv) [inline docs/docs/user-guide/examples/bionemo-esm2/inference.ipynb:9]: Maybe "out of sync"?

File: docs/docs/user-guide/examples/bionemo-esm2/mutant-design.ipynb [hunk 3]
```diff
@@ -7,7 +7,7 @@
    "source": [
     "[![ Click here to deploy.](https://uohmivykqgnnbiouffke.supabase.co/storage/v1/object/public/landingpage/brevdeploynavy.svg)](https://console.brev.dev/launchable/deploy?launchableID=env-2p81OcxdHAJZLSOc2KW14WW1YU8)\n",
     "\n",
-    "<div class=\"alert alert-block alert-info\"> <b>NOTE</b> It takes about 10 minutes to deploy this notebook as a Launchable. As of this writing, we are working on a free tier so a credit card may be required. You can reach out to your NVIDIA rep for credits.</div>"
+    "<div class=\"alert alert-block alert-info\"> <b>NOTE</b> It takes about 10 minutes to deploy this notebook as a Launchable. As of this writing, we are working on a free tier so a credit card may be required. You can reach out to your NVIDIA rep for credits. (Note: This links to the nightly release and may be out of date with these docs.) </div>"
```
Comment (tshimko-nv) [inline docs/docs/user-guide/examples/bionemo-esm2/mutant-design.ipynb:10]: Same, "out of sync"?

File: docs/docs/user-guide/examples/bionemo-esm2/pretrain.md [hunk 4]
```diff
@@ -8,7 +8,11 @@ The ESM-2 model is a transformer-based protein language model that was pretraine
...
+All commands should be executed inside the BioNeMo docker container, which has all ESM-2 dependencies pre-installed. The BioNeMo Framework container can run in a brev.dev launchable: [![ Click here to deploy.](https://uohmivykqgnnbiouffke.supabase.co/storage/v1/object/public/landingpage/brevdeploynavy.svg)](https://console.brev.dev/launchable/deploy/now?launchableID=env-2pPDA4sJyTuFf3KsCv5KWRbuVlU). It takes about 10 minutes to deploy this notebook as a Launchable. As of this writing, we are working on a free tier so a credit card may be required. You can reach out to your NVIDIA rep for credit. After launching the instance, launch a Terminal session in the jupyter lab UI. (Note: This links to the nightly release and may be out of date with these docs.)
```
Comment (tshimko-nv) [inline docs/docs/user-guide/examples/bionemo-esm2/pretrain.md:11]: Jupyter Lab <- Capitalization

File: docs/docs/user-guide/getting-started/access-startup.md [hunk 5]
```diff
@@ -11,11 +11,13 @@ BioNeMo Framework and begin exploring its features and capabilities.
 
 ## Access the BioNeMo Framework
 
-To access the BioNeMo Framework container, you will need a free NVIDIA GPU Cloud (NGC) account and an API key linked to
-that account.
+### Brev.Dev Access
+The BioNeMo Framework container can run in a brev.dev launchable: [![ Click here to deploy.](https://uohmivykqgnnbiouffke.supabase.co/storage/v1/object/public/landingpage/brevdeploynavy.svg)](https://console.brev.dev/launchable/deploy/now?launchableID=env-2pPDA4sJyTuFf3KsCv5KWRbuVlU). It takes about 10 minutes to deploy this notebook as a Launchable. As of this writing, we are working on a free tier so a credit card may be required. You can reach out to your NVIDIA rep for credit. After launching the instance, launch a Terminal session in the jupyter lab UI. (Note: This links to the nightly release and may be out of date with these docs.)
```
Comment (tshimko-nv) [inline docs/docs/user-guide/getting-started/access-startup.md:15]: "out of sync" again?

File: docs/docs/user-guide/getting-started/access-startup.md [hunk 6]
```diff
@@ -11,11 +11,13 @@ BioNeMo Framework and begin exploring its features and capabilities.
...
 
 ### NGC Account and API Key Configuration
 
+Another option to access the BioNeMo Framework container is to use a free NVIDIA GPU Cloud (NGC) account and an API key linked to
```
Comment (tshimko-nv) [inline docs/docs/user-guide/getting-started/access-startup.md:19]: This seems unfinished?

**Review by tshimko-nv — APPROVED** [review]
Some light comments, mostly on wording. Otherwise, one sentence looks truncated. Good once that's fixed. Thanks!

**Pull request merged**